pep/app/pods/most-cited-loading/route.ts

```diff
@@ -1,7 +1,10 @@
 import Route from '@ember/routing/route';
+
 import { PageNav } from 'pep/mixins/page-layout';
 
 export default class MostCitedLoading extends PageNav(Route) {
     navController = 'most-cited';
-    navTemplate = 'most-cited/nav';
+    get navTemplate() {
+        return 'most-cited/nav';
+    }
```

Comment:
why is this needed for navTemplate but not navController?

Reply:
It's due to how navTemplate is defined in the function. `navController` is a property and `navTemplate` is a getter